pkg/fftm/route__root_command.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fftm
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/hyperledger/firefly-common/pkg/ffapi"
+	"github.com/hyperledger/firefly-common/pkg/i18n"
+	"github.com/hyperledger/firefly-transaction-manager/internal/tmmsgs"
+	"github.com/hyperledger/firefly-transaction-manager/pkg/apitypes"
+	"github.com/hyperledger/firefly-transaction-manager/pkg/ffcapi"
+)
+
+var postRootCommand = func(m *manager) *ffapi.Route {
+	return &ffapi.Route{
+		Name:            "postRootCommand",
+		Path:            "/",
+		Method:          http.MethodPost,
+		PathParams:      nil,
+		QueryParams:     nil,
+		Description:     tmmsgs.APIEndpointPostSubscriptions,
```

Comment:
This doesn't look like the correct description for this route

Reply:
Also added `deployRequest` to to the input schema anyOf definition